balances/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["encointer.org <alain@encointer.org> and Parity Technologies <admin@p
 edition = "2021"
 
 [dependencies]
+approx = "0.5.0"
 codec = { package = "parity-scale-codec", version = "2.3.1", default-features = false, features = ["derive"] }
```

Comment:
As `approx` is only used in the benchmarks, it should be an optional dependency and enable in with the `benchmark` feature.